### PR TITLE
Fix jutsu page null check on ninjutsu

### PR DESCRIPTION
**Pull request by aumu1031**
Disregard stroke in commit title

### CODE REVIEW

**Comment by aumu1031** [comment]
Basic meme selector functionality in second commit

File: pages/jutsu.php [hunk 1]
```diff
@@ -188,10 +188,12 @@ function jutsu(): void {
 
     if($jutsu_to_view == null) {
         $sortedJutsu = [];
-        foreach($player->ninjutsu_ids as $jutsu_id) {
-            $sortedJutsu[] = $player->jutsu[$jutsu_id]->rank;
+        if ($player->ninjutsu_ids) {
+            foreach($player->ninjutsu_ids as $jutsu_id) {
+                $sortedJutsu[] = $player->jutsu[$jutsu_id]->rank;
+            }
+            array_multisort($sortedJutsu, $player->ninjutsu_ids);
```
Comment (levimeahan) [inline pages/jutsu.php:195]: Hmm... What if we go the other way with this and change the definition to initialize `ninjutsu_ids` (and tai/gen) as empty arrays? That seems simpler to use, and we can still check `count($player->ninjutsu_ids)` if desired

File: templates/chat.php [hunk 2]
```diff
@@ -102,6 +121,15 @@
...
         <tr>
             <td style="text-align: center;">
+                <button id="meme_toggle">+ Meme</button>
+                <div id="meme_box" class="hidden">
+                    <?php $memes = $system->getMemes(); ?>
+                    <?php for ($i = 0; $i < count($memes["codes"]); $i++) { ?>
+                    <div data-code="<?=$memes["codes"][$i]?>" class="meme_select">
+                        <?=$memes["images"][$i]?>
+                    </div>
+                    <?php }?>
```
Comment (levimeahan) [inline templates/chat.php:131]:
```suggestion
                    <?php for ($i = 0; $i < count($memes["codes"]); $i++): ?>
                    <div data-code="<?=$memes["codes"][$i]?>" class="meme_select">
                        <?=$memes["images"][$i]?>
                    </div>
                    <?php endfor; ?>
```

Comment (levimeahan) [reply templates/chat.php:131]: Template syntax makes this a little cleaner

File: pages/jutsu.php [hunk 3]
```diff
@@ -188,10 +188,12 @@ function jutsu(): void {
...
         }
```
Comment (levimeahan) [inline pages/jutsu.php:196]: We can actually delete this whole block, not needed

File: pages/jutsu.php [hunk 4]
```diff
@@ -188,10 +188,12 @@ function jutsu(): void {
...
-        array_multisort($sortedJutsu, $player->ninjutsu_ids);
     }
```
Comment (levimeahan) [inline pages/jutsu.php:197]: We can actually delete this whole block, not needed

Comment (aumu1031) [reply pages/jutsu.php]: Deleted

**Review by levimeahan — APPROVED** [review]

**Pull request merged**